libs/stats/odc/stats/model.py

```diff
@@ -146,6 +154,15 @@ class OutputProduct:
     href: str = ""
     region_code_format: str = "x{x:02d}y{y:02d}"
     cfg: Any = None
+    naming_conventions_values: str = "dea_c3"
+    explorer_path: str = "https://explorer.dea.ga.gov.au/"
+    inherit_skip_properties: Optional[List[str]] = None
+    preview_image: Optional[List[Any]] = None
+    preview_image_singleband: Optional[List[Any]] = None
+    classifier: str = "level3"
+    maturity: str = "final"
+    collection_number: int = 3
+    nodata: int = -999
```

Comment:
`nodata` is per band, not all outputs have the same `nodata` across al bands

Reply:
You are right. I will move the nodata setting to band level in the config file.